Framework/Source/Graphics/RenderGraph/RenderGraph.h

```diff
@@ -134,7 +134,7 @@ namespace Falcor
 
         /** Get all output names for the render graph
         */
-        std::vector<std::string> getAllOutputs() const;
+        std::vector<std::pair <std::string, bool> > getAvailableOutputs() const;
```

Comment:
I don't like pairs, it's unclear what each element means. Replace this with a struct please